Controllers/UsersController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using awsomAPI.Models;
+
+namespace awsomAPI.Controllers
+{
+    [Route("/")]
+    [ApiController]
+    public class RolesController : ControllerBase
+    {
+        private readonly AwsomApiContext _context;
+
+        public RolesController(AwsomApiContext context)
+        {
+            _context = context;
+        }
+        
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<User>>> GetUsers()
+        {
+            return await _context.Users.ToListAsync();
+        }
+
+        [HttpGet("{id}")]
+        public async Task<ActionResult<User>> GetUser(long id)
+        {
+            var user = await _context.Users.FindAsync(id);
+
+            if(user == null) {
+              return NotFound();
+            }
+            return user;
+        }
+
+        [HttpPost]
+        public void Post()

```

Comment:
Why is this empty?

Reply:
I think I know what you are trying to do, but let's try to avoid committing empty methods. Unless for some reason .net core needs them?

Reply:
It was just part of scaffolding. I can remove the empty ones until we build them.